### DISCUSSION TITLE
Is it possible to set up a transfer with different outgoing and incoming dates?

### DISCUSSION THREAD

**Discussion by baflo** [firefly-iii/firefly-iii#5341] [Firefly III] [answered]
Hi,

I cannot find information on this and I've been trying with a lot of settings, but could not make it work. My intention is to make a tranfer appear in the target account one or two days after the source account. 

This is just to resemble the actual money flow from one bank to the other.

Cheers
Florian

**Comment by JC5** [comment, marked as answer]
This isn't possible I'm afraid.

**Reply by baflo** [reply]
Ok, at least good to know. Thanks :)
